### ISSUE TITLE
[Adaptive Cards-Home]: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds for the links.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9193]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Win11
OS Version: 24H2 (OS Build 22631.5189)
URL: https://adaptivecards.io/
Edge dev Browser version: 137.0.3255.0 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on edge dev browser.
2. Home page will appear.
3. Run the tool and observe the issue.

**Actual Result:**
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.w3-large[target="_blank"][rel="noopener"]:nth-child(1)).
Currently luminosity Ratio for the link text with background is 4.25 which is less than 4.5:1.

**Expected Result:**
Luminosity ratio for the link text with background should be equal to or greater than 4.5:1.

**Element path:**
.w3-large[target="_blank"][rel="noopener"]:nth-child(1)

**Snippet:**
<a class="w3-large" target="_blank" rel="noopener" href="https://adaptivecards.microsoft.com">Adaptive Card Documentation Hub</a>

**How to fix:** 
Fix any of the following:
  Element has insufficient color contrast of 4.25 (foreground color: #0b78d0, background color: #fdf5fc, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1

**User Impact:**
It will impact low vision user as they will not able to see the link text properly if the CCA is less than required contrast.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

### Screenshots

![Image](https://github.com/user-attachments/assets/cb544003-19e2-4e15-a03b-a7073e160c2c)

![Image](https://github.com/user-attachments/assets/7b58394a-3bda-49b7-ac8d-48c0b91e05b5)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is now working as expected, hence closing this issue.

<img width="1920" height="1200" alt="Image" src="https://github.com/user-attachments/assets/aaeb7d62-30e3-461d-ac1d-eb15c48711b5" />
